aws/lambda/pytorch-auto-revert/Makefile

```diff
@@ -1,23 +1,40 @@
 all: run-local
 
+.PHONY: clean
 clean:
 	rm -rf deployment
-	rm -rf venv
 	rm -rf deployment.zip
+	rm -rf venv
+	# it makes sense for this to be the last one, or at least after rm -rf venv
+	find . -name __pycache__ -type d | xargs rm -rf
 
 venv/bin/python:
 	virtualenv venv
 	venv/bin/pip install -r requirements.txt
 
+venv/bin/lintrunner: venv/bin/python
+	venv/bin/pip install lintrunner==0.12.5 boto3-stubs==1.34.51
+	# lintrunner only works properly with virtualenv if you activate it first
+	. venv/bin/activate && lintrunner init --config ../../../.lintrunner.toml
+
 .PHONY: run-local
 run-local: venv/bin/python
-	venv/bin/python auto_revert.py --max-hours 70 --worker-pool-size 8  # --rebuild-table
+	venv/bin/python -m pytorch_auto_revert
+
+.PHONY: run-local-workflows
+run-local-workflows: venv/bin/python
+	venv/bin/python -m pytorch_auto_revert workflows pull.yml
 
 deployment.zip:
 	mkdir -p deployment
-	cp auto_revert.py ./deployment/.
+	cp -a pytorch-auto-revert. ./deployment/.
```

Comment:
does this work? doesn't have to be `pytorch-auto-revert/` or  ?

Reply:
ohh good catch, used '-' over '_' for the folder name :)